lib/Installation/ModuleRegistration/ModuleRegCodePage.pm

```diff
@@ -23,18 +23,35 @@ sub new {
 sub init {
     my ($self) = @_;
     $self->SUPER::init();
-    $self->{tb_we_code} = $self->{app}->textbox({id => '"sle-we"'});
+    $self->{txb_we_code} = $self->{app}->textbox({id => '"sle-we"'});
```

Comment:
you can also drop `new()` in this Page for the same reason.

Reply:
yes, done.